docker.md

```diff
@@ -4,6 +4,23 @@ To build the docker image, run the following command:
 
 `docker build -t "arc:latest" .`
 
-To run the docker image, run the following command:
+To run the docker image for testing, run the following command:
 
-`docker run -p 3333:3333 -e ARCOS_LISTEN_CONFIG_OVERRIDE="true" arc:latest`
\ No newline at end of file
+`docker run -p 3333:3333 -e ARCOS_LISTEN_CONFIG_OVERRIDE="true" arc:latest`
+
+In production, use Docker Compose. Here is an example `docker-compose.yml` file:
+
+```yaml
+version: '3.7'
+services:
+  arc:
+    image: arc:latest
+    ports:
+      - 3333:3333
+    environment:
+      - ARCOS_LISTEN_CONFIG_OVERRIDE=true
+    volumes:
+      - ./data:/ArcOS/data
+```
+
+Then run `docker-compose up` to start the container.
```

Comment:
Any reason for not using -d in the docker compose? [docker-compose is also deprecated and has been replaced with docker compose. ](https://docs.docker.com/compose/migrate/).

Can be changed to docker compose up -d and should work the same whilst also running in background mode as opposed to foreground mode which is how most people would have it anyways.

Reply:
Fixed